### PR TITLE
Add map_type and map_size quests

### PR DESCRIPTION
**Pull request by wielandb**
This PR adds two quests:

- What kind of map is this?
- What area is covered by this map?

The values are taken from the wiki pages. 

Would like thoughts/comments on the following things:
- Do we need custom icons for the quests? Or is the `board_type` icon okay?
- Are the descriptions of the answers okay? Or do we need pictures for understandability?

photo_2023-11-02_21-10-51
photo_2023-11-02_21-10-53

### CODE REVIEW

**Comment by wielandb** [comment]
Added quest icons.

**Comment by wielandb** [comment]
The wiki doesn't really provide images we could use. I would need to find some examples myself if the images are deemed _neccecary_ for merge of this PR.

**Comment by wielandb** [comment]
Just as a heads-up, after adding the icons, the app builds succesfully but crashes on startup. I have no idea how I did this since I only added the icons, but I would not consider this PR "tested" anymore.

**Comment by Helium314** [comment]
> Or do we need pictures for understandability?

I think for `map_size` descriptions are enough, but images would be helpful for `map_type`.

> crashes on startup

It doesn't look like there is an error in your last commit. What does the log say?

**Comment by wielandb** [comment]
> I think for map_size descriptions are enough, but images would be helpful for map_type.

Okay, I'll be on the hunt for some example pictures.

> It doesn't look like there is an error in your last commit. What does the log say?

I found the problem. When pulling the latest version from upstream, a new quest with the same ordinal I used was pulled in. Fixed that.

**Comment by wielandb** [comment]
Added images.
photo_2023-11-09_11-07-15

**Comment by wielandb** [comment]
One images was taken from the OSM Wiki and is licensed under CC-BY. Do we have a file where we name the authors in SCEE?

**Comment by wielandb** [comment]
> I've added some missing ones here: https://wiki.openstreetmap.org/wiki/Tag:information%3Dmap

Oh theese are way better than mine! I think I'll use them.

**Comment by wielandb** [comment]
Images replaced and licenses added. 
Should be ready for review now.

**Comment by Helium314** [comment]
I did not yet test  it, but some comments:
* could you convert the png images to jpg if there is a reasonable decrease in size?
* raster images should use [density qualifiers](https://developer.android.com/training/multiscreen/screendensities#TaskProvideAltBmp), i.e. use hdpi and similar folders for reduced size images, you can compare with images from other quests
* `map_type_1.png` is missing in authors.txt
* `MapType` appears to use placeholders, e.g. `ZERO("0")` instead of e.g. `TOPOGRAPHIC("tope")`

File: app/src/main/java/de/westnordost/streetcomplete/quests/map/AddMapTypeForm.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        MapType.SCHEME,
+        MapType.TOPOSCOPE
+    ).toItems()
+
+    override val itemsPerRow = 1
+    override val moveFavoritesToFront = false
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        imageSelector.cellLayoutId = R.layout.cell_labeled_icon_select_via_ferrata_scale
```
Comment (wielandb) [inline app/src/main/java/de/westnordost/streetcomplete/quests/map/AddMapTypeForm.kt:23]: I reuse the layout `R.layout.cell_labeled_icon_select_via_ferrata_scale` here. Is such reuse okay or should I create a copy of it?

Comment (Helium314) [reply app/src/main/java/de/westnordost/streetcomplete/quests/map/AddMapTypeForm.kt:23]: It's fine, no issue with that.

**Comment by wielandb** [comment]
> * could you convert the png images to jpg if there is a reasonable decrease in size?
> * raster images should use [density qualifiers](https://developer.android.com/training/multiscreen/screendensities#TaskProvideAltBmp), i.e. use hdpi and similar folders for reduced size images, you can compare with images from other quests
> * `map_type_1.png` is missing in authors.txt
> * `MapType` appears to use placeholders, e.g. `ZERO("0")` instead of e.g. `TOPOGRAPHIC("tope")`

Everything done.

**Pull request merged**